projects/packages/plans/src/class-current-plan.php

```diff
@@ -10,6 +10,7 @@
 namespace Automattic\Jetpack;
 
 use Automattic\Jetpack\Connection\Client;
+use Jetpack_Options;
```

Comment:
Are you using this anywhere?

Reply:
Yup, sorry I had forced pushed a few things to clean the PR and missed doing it correctly! All resolved now.